### PR TITLE
Scaffold: Run update-template

### PR DESCRIPTION
**Pull request by coreyfarrell**
Going to wait for this one to be reviewed before I post any more to make sure I did it right.  This PR has one commit that is just the result of `@gulpjs/update-template`, the second commit fixes a linter complaint in index.js as well as fixing tests to work with the newer version of expect.

Not sure the best for commit messages, this is a breaking change as it officially drops node.js < 10.13.0.

### CODE REVIEW

**Comment by phated** [comment]
I've been using this commit message:
> Breaking: Upgrade scaffold, dropping node <10 support

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2017 Blaine Bublitz <blaine.bublitz@gmail.com>, Eric Schoffstall <yo@contra.io> and other contributors
+Copyright (c) 2020 Blaine Bublitz <blaine.bublitz@gmail.com>, Eric Schoffstall <yo@contra.io> and other contributors
```
Comment (phated) [inline LICENSE]: I believe this is supposed to be 2017, 2020
```suggestion
Copyright (c) 2017, 2020 Blaine Bublitz <blaine.bublitz@gmail.com>, Eric Schoffstall <yo@contra.io> and other contributors
```

**Comment by coreyfarrell** [comment]
Single commit for scaffold update and post-fixes or separate?

**Review by phated — CHANGES_REQUESTED** [review]
Just 1 nit, but this looks amazing. I'm super happy someone besides me was able to upgrade the scaffolds using the tooling 🎊

**Comment by phated** [comment]
> Single commit for scaffold update and post-fixes or separate?

Either is fine. I can also squash it into 1 when merging if you don't want to bother.

**Review by phated — APPROVED** [review]
🎉

**Comment by coreyfarrell** [comment]
CI isn't running.  Does it have to be merged before that works?

**Comment by phated** [comment]
Yeah, GH Actions won't run if you submit the first GH Actions stuff from a fork. Do you have the access you need to push branches directly to gulp stuff?

**Comment by phated** [comment]
Landed from #8

**Pull request closed without merging**